source/partials/_header.erb

```diff
@@ -33,7 +33,17 @@
             </div>
           </div>
         </li>
-	      <li class="nav-item <%= nav_active("community.html") %>"><a href="/community">Community</a></li>
+	      <li class="nav-item dropdown">
+          <a class="dropdown-link" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Community</a>
+          <div class="dropdown-menu dropdown-menu--multi expandable">
+            <div class="dropdown-expand">
+              <div class="dropdown-expand__list">
+                <a class="dropdown-item" href="/community">Solidus Community</a>
```

Comment:
I can't find an alternative at the moment but it's quite odd to see `Solidus Community` into `Community`. We need to find something else here. I'm also ok splitting the page if that helps (maybe we can split in Contributors and Governance?).

Reply:
I agree here, splitting the page makes a lot of sense.

Reply:
Yes, I didn't have a better idea than repeat the word `Community` 😓 That page is a sort of _About us_ but it sounds weird to me using it. 
Another option is to move the Events into the `Developer` dropdown menu, near to the Guides link - at the top. 

Splitting the actual Community page could work but we should revise the content for both the sections/pages, IMO.

Reply:
@kennyadsl I've tried to split the Community page as you suggested:

> I can't find an alternative at the moment but it's quite odd to see `Solidus Community` into `Community`. We need to find something else here. I'm also ok splitting the page if that helps (maybe we can split in Contributors and Governance?).

What do you think? It is a good idea to split that page since it was too long to scroll. I've preferred the _Core Team_ word instead of _Governance_ that sounds too institutional, IMO.

Reply:
I like it! I'd make 2 changes:

1. Move Contributors before Core Team in the menu. I think it's more interesting for users.
2. Change the `Contributors` section's title in the `Contributors` page to `Code Contributors` to differentiate that section from the page itself. This way, we will have the `Contributors` page that includes `Donors` and `Code Contributors`.